### PR TITLE
Sol suits lore, new generic softsuit sprite.

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
What it says up there. Also tweaked the Sol hardsuit while I was at it.
New:
image
dreammaker_t1gesbzeUx
dreammaker_1wtA0c54oJ
dreammaker_qi9LwMluhR
vs
image
dreammaker_rBS7n5GcHt
dreammaker_f2Pj7xQtCp
dreammaker_yE8wnHWmcM

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Comment by WickedCybs** [comment]
Think it would be nice if oxygen tanks became valid for the softsuits storage slot as well.

**Comment by TheDanseMacabre** [comment]
> Think it would be nice if oxygen tanks became valid for the softsuits storage slot as well.

Forgot to do this

**Comment by SleepyGemmy** [comment]
don't forget inhands and icon sprites for the new softsuit.

**Comment by TheDanseMacabre** [comment]
> don't forget inhands and icon sprites for the new softsuit.

no

File: html/changelogs/dansemacabre-larp.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Sol voidsuit and hardsuits have lore like the other suits."
+  - rscadd: "New sprite for the generic softsuit. Generic softsuit can now carry large air tanks."
```
Comment (Arrow768) [inline html/changelogs/dansemacabre-larp.yml:42]: Two changes on one line

**Review by Arrow768 — DISMISSED** [review]
If you have multiple changes, then these changes should be on their own lines in the changelog.
Please post a comparison picture of the old/new sprites.
The icon diffbot isnt great for reviewing multi-part sprites.

**Review by JohnWildkins — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**